src/vegarando.js

```diff
@@ -38,38 +61,58 @@ function prepareStyle() {
         box-shadow: inset 2rem 0px 0px 0px #7bc043;
         padding-left: 2rem;
      }
+     .highlight-good {
+        text-shadow: 0px 0px 5px rgb(0,255,0);
+     }
+     .highlight-bad {
+        text-shadow: 0px 0px 5px rgb(255,0,0);
+     }
     `;
 
     document.head.appendChild(styleTag);
 }
 
+function tagMeal(elem, classTag) {
+    elem.classList.add(classTag);
+    highlightWords(elem);
+}
+
 function tagMeals() {
     for (let c of document.getElementsByClassName("meal-container")) {
-        let mealName = c.getElementsByClassName("meal-name")[0]
-        let mealInfo = c.getElementsByClassName("meal__description-additional-info")[0];
-        
+        let mealName = c.getElementsByClassName("meal-name")[0];
+        let mealInfo = c.getElementsByClassName(
+            "meal__description-additional-info"
+        )[0];
```

Comment:
Relying on element 0 to exist is risky. If it doesn't script execution will halt completely at this point. Obviously the case is unlikely but worth considering in case they change markup.

Reply:
das ist leider nur durch den code format gekommen -> refactor iwann

Reply:
-> #9